ui/buildServer.sh

```diff
@@ -2,7 +2,5 @@
 
 set -e
 
-cd ..
-export addDevArtifacts=true
-./sbtwrapper "set ThisBuild / packageDoc / publishArtifact := false; set Compile / doc / sources := Seq.empty; dist/Universal/stage"
-cd -
+cd "$(dirname $0)"/..
```

Comment:
`$0` should be quoted, or this will break when path contains space character. There are also a few more shell scripts where this was omitted.